plugins/sql_config_validator.h

```diff
@@ -98,6 +111,14 @@ template<typename FieldType> struct NoGroupByHaving
     }
 };
 
+template<typename FieldType> struct NoGroupByHaving<Optional<FieldType> >
```

Comment:
This is more of a c++ question I guess but can't the Optional be templated away in the other NoGroupByHaving at line 99? Meaning I could declare a NoGroupByHaving<Optional<bla>> and the FieldType would take the type Optional<bla> ?